src/main.rs

```diff
@@ -35,3 +38,25 @@ fn main() -> wry::Result<()> {
         Ok(())
     }
 }
+
+fn parse_command_parameters() -> CommandParameters {
+    let mut default_config_path: Option<PathBuf> = None;
+    let mut address: Option<String> = None;
+    for string in env::args() {
+        if string.starts_with("http") {
```

Comment:
Hurgh, what about [clap-derive](https://docs.rs/clap/latest/clap/_derive/index.html#overview)?

Reply:
Never heard of it. I'll look into it

Reply:
I second @flokli, let's see what this'll look like with derivations.